lib/codelog/command/step/version.rb

```diff
@@ -28,13 +27,24 @@ def run
           abort(Codelog::Message::Error.missing_version_number) if @version.nil?
           abort(Codelog::Message::Error.already_existing_version(@version)) if version_exists?
           abort(Codelog::Message::Error.no_detected_changes(@version)) unless unreleased_changes?
-          chdir Dir.pwd do
-            create_version_changelog_from changes_hash
-          end
+
+          generate_changelog_content_from(changes_hash)
```

Comment:
@emilianoLeite @rmohashi If you take a look on the 'outputters', you'll see that they have different interfaces. Because of that, i think using dependency injection to choose the output was not the best sollution, so, i thought was a good idea to make this class have the responsability of only gathering the content of the next release, and spread its other prevous responsabilities elsewhere, like output choosing or even the output logic itself.